threadservice/options.go

```diff
@@ -114,29 +114,29 @@ func PutOptions(opts ...PutOption) *PutSettings {
 	return options
 }
 
-// ListenOpt is an instance helper for creating listen options.
-var ListenOpt ListenOption
+// SubOpt is an instance helper for creating subscription options.
+var SubOpt SubOption
```

Comment:
Is this used for something? (same with `AddOpt`)

Reply:
These are just helper for adding options: https://github.com/textileio/go-textile-threads/blob/master/test/threads_suite.go#L97. There very well could be a better way to handle options in general